test/data_structure/bitmap/check_bitmap.c

```diff
@@ -53,7 +54,7 @@ START_TEST(test_bitset_getset)
         }
     }
 
-    for (j=0; j < 4; j++) { /* set these bits back to 0 */
+    for (j=0; j < cols_size; j++) { /* set these bits back to 0 */
```

Comment:
nit: can you also fix the spacing in this loop? to `for (j = 0; ...`

Reply:
Done.